docs/explanations/how_to_interpret_output_of_pro_cve.rst

```diff
@@ -0,0 +1,124 @@
+How to interpret the output of pro cve
```

Comment:
Shouldn't all of this content be in the CLI reference page? (same for the second page as well)

Reply:
I don't think all of them, due to the size of it. But I have added the missing entries on the CLI reference page